static/js/containers/ProfileFormContainer.js

```diff
@@ -112,15 +113,19 @@ class ProfileFormContainer extends React.Component {
     dispatch(startProfileEdit(username));
   };
 
-  saveProfile(isEdit: boolean, validator: Validator|UIValidator, profile: Profile, ui: UIState) {
+  saveProfile(isEdit: boolean, validator: Validator|UIValidator|null, profile: Profile, ui: UIState) {
```

Comment:
An example of such a simple validator is the `profileImageValidation` function I implemented recently (in `lib/validation/profile.js`)